modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -65,6 +65,7 @@ def generate_show_output(bgs_claim:, lighthouse_claim:) # rubocop:disable Metric
                                               lighthouse_id: nil,
                                               upstream_id: bgs_details[:benefit_claim_id])
           end
+          structure.merge!(tracked_items: map_bgs_tracked_items(bgs_claim))
```

Comment:
In the spirit of keeping alphabetical consistency when possible, I'd move `tracked_items` down below `supporting_documents`.